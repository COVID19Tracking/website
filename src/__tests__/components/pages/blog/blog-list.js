import React from 'react'
import renderer from 'react-test-renderer'
import BlogList from '../../../../components/pages/blog/blog-list'

const blogItems = [
  {
    node: {
      id: 'a',
      slug: 'a',
      title: 'Blog list item',
      author: {
        name: 'test name',
        twitterLink: 'https://twitter.com/test',
      },
      publishDate: 'August 1, 20202',
    },
  },
]

describe('Components : Pages : Blog : List', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<BlogList items={blogItems} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
