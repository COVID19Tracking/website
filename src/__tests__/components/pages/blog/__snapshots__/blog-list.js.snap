// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Blog : List renders correctly 1`] = `
<ul
  className="press-list pressList"
>
  <li>
    <cite
      className="title"
    >
      <a
        href="/blog/a"
      >
        Blog list item
      </a>
    </cite>
    <div
      className="detail-text detailText"
    >
      test name
      <span
        className="dotSeparator"
      >
        â€¢
      </span>
      August 1, 20202
    </div>
  </li>
</ul>
`;
